rllib/policy/sample_batch.py

```diff
@@ -346,42 +347,47 @@ def split_by_episode(self) -> List["SampleBatch"]:
 
         Examples:
             >>> from ray.rllib.policy.sample_batch import SampleBatch
+            >>> # "eps_id" is present
             >>> batch = SampleBatch( # doctest: +SKIP
             ...     {"a": [1, 2, 3], "eps_id": [0, 0, 1]})
             >>> print(batch.split_by_episode()) # doctest: +SKIP
             [{"a": [1, 2], "eps_id": [0, 0]}, {"a": [3], "eps_id": [1]}]
+            >>>
+            >>> # "eps_id" not present, split by "dones" instead
```

Comment:
will you error out if done is not True at all or for the last episode? What's the behavior then?